src/sprite_battler.cpp

```diff
@@ -144,10 +144,6 @@ void Sprite_Battler::Update() {
 			}
 		}
 	}
-
-	SetX(battler->GetDisplayX());
-	SetY(battler->GetDisplayY());
-	SetZ(battler->GetBattleY());
```

Comment:
I think I found a use for these lines: The purpose was probably (y u no comment) to support RPG2k3 "Animation 2" stuff that contains e.g. "Move to enemy". But we don't implement this by now so no damage done.

Reply:
I am currently working on something in sprite_battler, history says these were added for screen shaking (891631fc4f29).

Reply:
They've been there since the file was first added in c6f41c7. 891631f just changed `GetBattleX` to `GetDisplayX` to account for shaking.

Reply:
We were talking about the added ones in `Update()` ;-)